### ISSUE TITLE
I am unable to customize GitHub projects on the integration settings

### ISSUE THREAD

**Issue by simplytammy** [thepracticaldev/dev.to#8790]
**Describe the bug**

I tried changing the GitHub projects visible on my profile, but it failed to load my  repos despite being signed in using GitHub

**To Reproduce**

1. Go to settings
2. Click on integrations
3. An error  is displayed after a few seconds

**Expected behavior**
Not sure what the expected behavior should be, but I expected to see a list of my public repos to choose from 

**Screenshots**
![dev-int](https://user-images.githubusercontent.com/36845308/85168590-7f452b00-b262-11ea-8633-48b732b5aa4c.png)

- OS: Linux Mint 19.2
- Browser: Firefox
- Version: Firefox 73

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by citizen428** [comment]
@simplytammy This is a known problem, but alas it's intermittent and we haven't been able to pinpoint the exact problem yet.

**Comment by rhymes** [comment]
Closing this in favor of https://github.com/thepracticaldev/dev.to/issues/8790
